dotCMS/src/integration-test/java/com/dotcms/integritycheckers/ContentFileAssetIntegrityCheckerTest.java

```diff
@@ -117,4 +120,48 @@ public void TestFixConflictGeneratesContentletAsJson() throws Exception {
         Assert.assertTrue(validateFix(remoteIdentifier));
     }
 
+    /**
+     * Method to test: {@link ContentFileAssetIntegrityChecker#executeFix(String)}
+     * When: Tests that after conflicts are detected a fix is applied in favor of remote fileAsset.
+     * Should: Columns Asset_subtype, owner and create_date should be populated
+     * @throws Exception
+     */
+    @Test
+    public void test_execute_identifierColumnsNotNull() throws Exception {
+        final Host host = APILocator.getHostAPI().findDefaultHost(APILocator.systemUser(), false);
+        final Folder folder = new FolderDataGen().name("testFolder"+UUID.randomUUID().toString()).site(host).nextPersisted();
+        final Contentlet contentlet = FileAssetDataGen.createFileAsset(folder, "text1FileAsset"+UUID.randomUUID().toString(), ".txt");
+        final FileAsset fileAsset = APILocator.getFileAssetAPI().fromContentlet(contentlet);
+        final ContentFileAssetIntegrityChecker integrityChecker = new ContentFileAssetIntegrityChecker();
+        final Tuple2<String, String> remoteIdentifierAndInode = introduceConflict(fileAsset ,endpointId.get());
+        final String remoteIdentifier = remoteIdentifierAndInode._1();
+        final String remoteWorkingInode = remoteIdentifierAndInode._2();
+
+        integrityChecker.executeFix(endpointId.get());
+
+        final DotConnect dotConnect = new DotConnect();
```

Comment:
when you are using dotCOnnect on test, you should handle the connection in this case you have to close it at the end

Reply:
Added the requested changes https://github.com/dotCMS/core/pull/25325/commits/83fd7f33ff512c7e88b0f79d03bcda1bef3915c1